.github/workflows/maven-build-caching-samples-verification.yml

```diff
@@ -21,5 +23,11 @@ jobs:
       - name: Set up Yarn
         run: npm install -g yarn
       - name: Build with Maven
+        id: mvn-build
         working-directory: ./build-caching-maven-samples
-        run: ./mvnw clean verify -Dgradle.scan.disabled=true
+        run: |
+          ./mvnw clean verify -Dgradle.enterprise.url=https://ge.solutions-team.gradle.com -Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug 2>&1 | tee -a /tmp/gradle-build.log
```

Comment:
I would have the GE URL as Secret

Reply:
@jprinet the URL of the Solutions Team's GE instance is public. No need to hide it.